modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/EditFileEntryMVCActionCommand.java

```diff
@@ -1034,6 +1005,35 @@ private String _getSaveAndContinueRedirect(
 		return portletURL.toString();
 	}
 
+	private void _getUrlTitleChangedMessage(
+		ActionRequest actionRequest, String originalUrlTitle,
+		long fileEntryId) {
+
+		FriendlyURLEntry friendlyURLEntry =
+			_friendlyURLEntryLocalService.fetchMainFriendlyURLEntry(
+				_classNameLocalService.getClassNameId(FileEntry.class),
+				fileEntryId);
+
+		String currentUrlTitle = friendlyURLEntry.getUrlTitle();
+
+		if (Validator.isNotNull(originalUrlTitle) &&
+			!currentUrlTitle.equals(
+				_friendlyURLNormalizer.normalizeWithEncoding(
+					originalUrlTitle))) {
+
+			MultiSessionMessages.add(
+				actionRequest, "friendlyURLChanged",
```

Comment:
@adolfopa  Do you think that we must change `friendlyURLChanged` variable?

I think that on the message we are telling that we are changing the FriendlyURL so this can be a little confussing.

What do you think?

Reply:
Yes, I think this is fine as it is. But don't forget to comment in the description that you're doing that on purpose, so that Brian doesn't think you missed one occurrence.